spec/index.md

```diff
@@ -4141,6 +4159,44 @@ S with
 
 ```
 
+#### IC Management Canister: Metrics
+
+:::note
+
+The metrics management canister API is considered EXPERIMENTAL. Canister developers must be aware that the API may evolve in a non-backward-compatible way.
+
+:::
+
+The management canister returns metrics for a given subnet. The definition of the metrics values
+is not captured in this formal semantics.
+
+Conditions
+
+```html
+
+S.messages = Older_messages · CallMessage M · Younger_messages
+(M.queue = Unordered) or (∀ msg ∈ Older_messages. msg.queue ≠ M.queue)
+M.callee = ic_principal
+M.method_name = 'metrics'
+M.arg = candid(A)
+R = <implementation-specific>
+
+```
+
+State after
+
+```html
+
+S with
+    messages = Older_messages · Younger_messages ·
+      ResponseMessage {
+        origin = M.origin
+        response = Reply (candid(R))
+        refunded_cycles = M.transferred_cycles
+      }
+
+```
+
```

Comment:
@mraszyk, I'm not familiar with this notation. Do we need to update this part of the PR to reflect the discussions from last week and the description on the design slides, or are we good?

Reply:
The actual response is left under-specified
```
R = <implementation-specific>
```
in the formal semantics and thus no changes are required.